### PR TITLE
Support measuring of PMU firmware instructions/data, microcode patch and video binary interpreter

### PR DESCRIPTION
**Pull request by rihter007**

### CODE REVIEW

**Comment by rihter007** [comment]
> Maybe I am missing something, but I am having a hard time reproducing measurement for PMU firmware, for example. The source address and size match the metadata of the entry, but I cannot reproduce the hash. Have you tried to repro it, successfully?

Replied in PM

**Review by zaolin — APPROVED** [review]

**Comment by ChriMarMe** [comment]
When the conflicts have been resolved i provide approval

**Review by ChriMarMe — APPROVED** [review]

File: pkg/pcr/get_measurements_pcr0_amd.go [hunk 1]
```diff
@@ -96,20 +97,41 @@ func MeasurePSPVersion(image []byte, pspDirectoryLevel1, pspDirectoryLevel2 *amd
...
 		}
 
 		for _, entry := range biosDirectory.Entries {
-			if entry.Type == amd_manifest.BIOSRTMVolumeEntry {
-				return NewRangeMeasurement(MeasurementIDBIOSRTMVolume, entry.SourceAddress, uint64(entry.Size)), nil
+			if entry.Type == entryType {
+				foundEntries = append(foundEntries, entry)
 			}
 		}
+		break
```
[the remark this answers is not in the document]
Comment (xaionaro) [reply pkg/pcr/get_measurements_pcr0_amd.go:223]: Would be nice if in next PR you would add a comment :)

**Review by xaionaro — APPROVED** [review]
Approved. But it would be nice to add into the code comments explaining answers of all Marco's questions. It could be done in the next PR to avoid dropping approvals on this one :)

**Pull request merged**